test/unit-test/FreeRTOS_UDP_IP/FreeRTOS_UDP_IP_stubs.c

```diff
@@ -0,0 +1,154 @@
+/* Include Unity header */
+#include <unity.h>
+
+/* Include standard libraries */
+#include <stdlib.h>
+#include <string.h>
+#include <stdint.h>
+#include "FreeRTOS.h"
+#include "task.h"
+#include "list.h"
+
+#include "FreeRTOS_IP.h"
+#include "FreeRTOS_IP_Private.h"
+
+volatile BaseType_t xInsideInterrupt = pdFALSE;
+
+/** @brief The expected IP version and header length coded into the IP header itself. */
+#define ipIP_VERSION_AND_HEADER_LENGTH_BYTE    ( ( uint8_t ) 0x45 )
```

Comment:
These defines and variables do not seem to be used anywhere.

Reply:
Removed them. Had copied them from other unit-tests.